### ISSUE TITLE
[BUG] Window Switcher brings back the current window when pressing Command + Tab twice

### ISSUE THREAD

**Issue by Risinggoblin** [ejbills/DockDoor#363]
## Describe the bug
Pressing` CMD` + `Tab` twice  brings you back to the current window instead of switching to the second last window by recency.

## Steps to reproduce
Hold `Command `key and press `Tab` twice

## Expected behavior
The second last used window should become active - as it happens in Windows or when using AltTab (Mac app)

## Actual behavior
Current window remains in focus (effectively not doing anything at all)

## Screenshots
N/A

## Environment
 - macOS version: macOS 15.1 Sequoia
 - App version: 1.2.9

## Additional context
N/A

- [X] I have reviewed the open issues to ensure this is not a duplicate

**Comment by SarcevicAntonio** [comment]
Yeah I agree. I think this is mostly about the ordering of the windows in the window switcher and which index is the first focused window.

Right now it's like this:
1. Last Window (focused)
2. Current Window
3. Second To Last Window

But I expect it to be like this:
1. Current Window
2. Last Window (focused)
3. Second To Last Window

**Comment by ejbills** [comment]
![CleanShot 2024-12-21 at 21 55 45](https://github.com/user-attachments/assets/7f1a85c8-9ac3-4bc8-9a89-7530c1714ff6)
This will be a feature in the next version as pictured above!

**Comment by Risinggoblin** [comment]
Thanks for actioning on this. However, I think the current implementation still doesn't solve the issue. With the current implementation, pressing Command + Tab doesn't do anything at all. 

The ideal (and most natural - from usability and familiarity perspective) implementations should have the following behaviour:

* ```Cmd + Tab``` once : Last App
* ```CMD + Tab``` twice : Second Last App

**Comment by ejbills** [comment]
Hmm seems to be behaving inconsistently with the new option, I will look into this and include it in the next patch version.

**Comment by ejbills** [comment]
Will be in 1.6

**Comment by SarcevicAntonio** [comment]
I also really appreciate you picking this up 🙏🏼
Alt-Tab was starting to really break for me in latest Macos versions and some apps and with this I could make the switch, but I do think what @Risinggoblin and me referred would make moving around a bit more efficient.
